src/test/Bau.Test.Acceptance.CreateFile/Program.cs

```diff
@@ -13,7 +13,8 @@ static void Main(string[] args)
 
             if (args.Length == 0)
             {
-                throw new ArgumentException("No filename provided.");
+                Console.WriteLine("No filename provided.");
+                Environment.Exit(9999);

```

Comment:
The change to `.Exit` was required for my sanity. Using continuous testing tools and sometimes when running the acceptance tests I had to keep dismissing the dialog that results from unhanded exceptions.